rust/operator-binary/src/main.rs

```diff
@@ -5,7 +5,7 @@ use futures::stream::StreamExt;
 use stackable_operator::{
     YamlSchema,
     cli::{Command, ProductOperatorRun},
-    commons::authentication::AuthenticationClass,
+    crd::authentication::core as auth_core,
```

Comment:
I should have same the same (renamed to `auth_core`) in the other PRs. Will eventually do that.